common/src/crypto.rs

```diff
@@ -81,6 +81,73 @@ impl<'de, const N: usize> Deserialize<'de> for HexSerializedBytes<N> {
     }
 }
 
+#[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Hash)]
+pub struct HexSerializedVec {
```

Comment:
I think decoding function like ```decode_hex(s: &str) -> Result<Vec<u8>, ParseIntError>``` would be useful. For example, we may face many situations where we must directly convert a hex string to Vec<u8>.

Reply:
I think it's better to add such utility function after we actually find a use case.

Reply:
Ok